src/core/tritonserver.cc

```diff
@@ -1613,6 +1613,18 @@ TRITONSERVER_InferenceRequestAddRequestedOutput(
   return nullptr;  // Success
 }
 
+TRITONSERVER_Error*
```

Comment:
So here we have a real deficiency in the current API. There is no way to discover what output tensors are available. But we need an API that keeps ownership of the names with the InferenceRequest object so we don't have to worry about managing that. I suggest we add a TRITONSERVER_InferenceRequestOutputCount to return the number of output tensors (notice how we don't use "Get" in the API naming to be consistent with other existing API), and a TRITONSERVER_InferenceRequestOutputName(uint3_t index, const char** name) that returns the name of the output at that index.

Reply:
Done